src/app/components/AddSource.svelte

```diff
@@ -0,0 +1,183 @@
+<script lang="ts">
+  import worker from "../spawn-worker";
+  import Button from "./Button.svelte";
+  import SplitButton from "./SplitButton.svelte";
+  import ManualEntry from "./ManualEntry.svelte";
+  const UPLOAD_INPUT_ID = "upload-input";
+
+  const leftText = "UPLOAD";
+  const rightText = "DIRECT ENTRY";
+  export let addSource: boolean = false;
+  let manualEntry: boolean = false;
+  let onDraggedOver = false;
+  let fileList: any = [];
```

Comment:
would avoid using `any` if there is a type for the `fileList` array

Reply:
Sure, since fileList will be assigned to different data types, I change its type to File[]|FileList.

Reply:
If fileList could be assigned to different type, you could provide the known type as well as any. 
Example:
`let example: string | any`